src/styles/main.css

```diff
@@ -16,7 +16,9 @@
 }
 
 html {
+  background-color: black;
   font-size: 100%; /* 1rem = 16px */
+  color: white;
 }
```

Comment:
I think this will affect other pages, please double check other pages after merging the latest main

Reply:
removed "background-color: black"